src/praisonai/praisonai/cli/app.py

```
@@ -106,13 +110,387 @@ class OutputFormat(str, Enum):
+                _run_redo(args)
+            
+            return typer_get_command(standardise_app)
+        except Exception:
+            return None
+    
+    def _get_app_command(self) -> Optional[click.Command]:
+        """Get the app command."""
+        @app.command(name="app", context_settings={"allow_interspersed_args": False})
+        def app_cmd(
+            port: int = typer.Option(8000, "--port", "-p", help="Port to listen on"),
+            host: str = typer.Option("0.0.0.0", "--host", "-h", help="Host to bind to"),
```

Ruff (0.15.17)

[error] Possible binding to all interfaces

(S104)